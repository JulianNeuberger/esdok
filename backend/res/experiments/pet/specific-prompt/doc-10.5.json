{"text": "The MPON requests a device takeover bid of the MPOO . The MPOO sends a tender for the equipment takeover to the MPON . The MPON places an order at the MPOO . The MPOO confirms the order of the MPON and sends the master data .", "name": "doc-10.5", "id": "doc-10.5", "category": "", "tokens": [{"text": "The", "indexInDocument": 0, "posTag": "DT", "sentenceIndex": 0}, {"text": "MPON", "indexInDocument": 1, "posTag": "NNP", "sentenceIndex": 0}, {"text": "requests", "indexInDocument": 2, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "a", "indexInDocument": 3, "posTag": "DT", "sentenceIndex": 0}, {"text": "device", "indexInDocument": 4, "posTag": "JJ", "sentenceIndex": 0}, {"text": "takeover", "indexInDocument": 5, "posTag": "NN", "sentenceIndex": 0}, {"text": "bid", "indexInDocument": 6, "posTag": "NN", "sentenceIndex": 0}, {"text": "of", "indexInDocument": 7, "posTag": "IN", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 8, "posTag": "DT", "sentenceIndex": 0}, {"text": "MPOO", "indexInDocument": 9, "posTag": "NNP", "sentenceIndex": 0}, {"text": ".", "indexInDocument": 10, "posTag": ".", "sentenceIndex": 0}, {"text": "The", "indexInDocument": 11, "posTag": "DT", "sentenceIndex": 1}, {"text": "MPOO", "indexInDocument": 12, "posTag": "NNP", "sentenceIndex": 1}, {"text": "sends", "indexInDocument": 13, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "a", "indexInDocument": 14, "posTag": "DT", "sentenceIndex": 1}, {"text": "tender", "indexInDocument": 15, "posTag": "NN", "sentenceIndex": 1}, {"text": "for", "indexInDocument": 16, "posTag": "IN", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 17, "posTag": "DT", "sentenceIndex": 1}, {"text": "equipment", "indexInDocument": 18, "posTag": "NN", "sentenceIndex": 1}, {"text": "takeover", "indexInDocument": 19, "posTag": "NN", "sentenceIndex": 1}, {"text": "to", "indexInDocument": 20, "posTag": "TO", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 21, "posTag": "DT", "sentenceIndex": 1}, {"text": "MPON", "indexInDocument": 22, "posTag": "NNP", "sentenceIndex": 1}, {"text": ".", "indexInDocument": 23, "posTag": ".", "sentenceIndex": 1}, {"text": "The", "indexInDocument": 24, "posTag": "DT", "sentenceIndex": 2}, {"text": "MPON", "indexInDocument": 25, "posTag": "NNP", "sentenceIndex": 2}, {"text": "places", "indexInDocument": 26, "posTag": "NNS", "sentenceIndex": 2}, {"text": "an", "indexInDocument": 27, "posTag": "DT", "sentenceIndex": 2}, {"text": "order", "indexInDocument": 28, "posTag": "NN", "sentenceIndex": 2}, {"text": "at", "indexInDocument": 29, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 30, "posTag": "DT", "sentenceIndex": 2}, {"text": "MPOO", "indexInDocument": 31, "posTag": "NNP", "sentenceIndex": 2}, {"text": ".", "indexInDocument": 32, "posTag": ".", "sentenceIndex": 2}, {"text": "The", "indexInDocument": 33, "posTag": "DT", "sentenceIndex": 3}, {"text": "MPOO", "indexInDocument": 34, "posTag": "NNP", "sentenceIndex": 3}, {"text": "confirms", "indexInDocument": 35, "posTag": "VBZ", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 36, "posTag": "DT", "sentenceIndex": 3}, {"text": "order", "indexInDocument": 37, "posTag": "NN", "sentenceIndex": 3}, {"text": "of", "indexInDocument": 38, "posTag": "IN", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 39, "posTag": "DT", "sentenceIndex": 3}, {"text": "MPON", "indexInDocument": 40, "posTag": "NNP", "sentenceIndex": 3}, {"text": "and", "indexInDocument": 41, "posTag": "CC", "sentenceIndex": 3}, {"text": "sends", "indexInDocument": 42, "posTag": "VBZ", "sentenceIndex": 3}, {"text": "the", "indexInDocument": 43, "posTag": "DT", "sentenceIndex": 3}, {"text": "master", "indexInDocument": 44, "posTag": "NN", "sentenceIndex": 3}, {"text": "data", "indexInDocument": 45, "posTag": "NNS", "sentenceIndex": 3}, {"text": ".", "indexInDocument": 46, "posTag": ".", "sentenceIndex": 3}], "mentions": [{"type": "activity", "tokenDocumentIndices": [2]}, {"type": "activity", "tokenDocumentIndices": [13, 14, 15]}, {"type": "activity", "tokenDocumentIndices": [26, 27, 28]}, {"type": "activity", "tokenDocumentIndices": [35, 36, 37]}, {"type": "activity", "tokenDocumentIndices": [42, 43, 44, 45]}, {"type": "actor", "tokenDocumentIndices": [0, 1]}, {"type": "actor", "tokenDocumentIndices": [11, 12]}, {"type": "actor", "tokenDocumentIndices": [24, 25]}, {"type": "actor", "tokenDocumentIndices": [33, 34]}, {"type": "actor", "tokenDocumentIndices": [39, 40]}, {"type": "activity data", "tokenDocumentIndices": [3, 4, 5, 6]}, {"type": "activity data", "tokenDocumentIndices": [14, 15]}, {"type": "activity data", "tokenDocumentIndices": [27, 28]}, {"type": "activity data", "tokenDocumentIndices": [36, 37]}, {"type": "activity data", "tokenDocumentIndices": [43, 44, 45]}, {"type": "further specification", "tokenDocumentIndices": [3, 4, 5, 6]}, {"type": "further specification", "tokenDocumentIndices": [14, 15]}, {"type": "further specification", "tokenDocumentIndices": [27, 28]}, {"type": "further specification", "tokenDocumentIndices": [36, 37]}, {"type": "further specification", "tokenDocumentIndices": [43, 44, 45]}], "entities": [{"mentionIndices": [5, 7, 9]}, {"mentionIndices": [6, 8]}, {"mentionIndices": [10, 12, 13]}, {"mentionIndices": [0]}, {"mentionIndices": [1]}, {"mentionIndices": [2]}, {"mentionIndices": [3]}, {"mentionIndices": [4]}, {"mentionIndices": [11]}, {"mentionIndices": [14]}, {"mentionIndices": [15]}, {"mentionIndices": [16]}, {"mentionIndices": [17]}, {"mentionIndices": [18]}, {"mentionIndices": [19]}], "relations": [{"headMentionIndex": 0, "tailMentionIndex": 1, "type": "flow"}, {"headMentionIndex": 1, "tailMentionIndex": 2, "type": "flow"}, {"headMentionIndex": 2, "tailMentionIndex": 3, "type": "flow"}, {"headMentionIndex": 3, "tailMentionIndex": 4, "type": "flow"}, {"headMentionIndex": 0, "tailMentionIndex": 5, "type": "actor performer"}, {"headMentionIndex": 0, "tailMentionIndex": 15, "type": "further specification"}, {"headMentionIndex": 0, "tailMentionIndex": 10, "type": "uses"}, {"headMentionIndex": 1, "tailMentionIndex": 6, "type": "actor performer"}, {"headMentionIndex": 1, "tailMentionIndex": 16, "type": "further specification"}, {"headMentionIndex": 1, "tailMentionIndex": 11, "type": "uses"}, {"headMentionIndex": 1, "tailMentionIndex": 5, "type": "actor recipient"}, {"headMentionIndex": 2, "tailMentionIndex": 7, "type": "actor performer"}, {"headMentionIndex": 2, "tailMentionIndex": 17, "type": "further specification"}, {"headMentionIndex": 2, "tailMentionIndex": 12, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 8, "type": "actor recipient"}, {"headMentionIndex": 3, "tailMentionIndex": 8, "type": "actor performer"}, {"headMentionIndex": 3, "tailMentionIndex": 18, "type": "further specification"}, {"headMentionIndex": 3, "tailMentionIndex": 13, "type": "uses"}, {"headMentionIndex": 3, "tailMentionIndex": 7, "type": "actor recipient"}, {"headMentionIndex": 4, "tailMentionIndex": 8, "type": "actor performer"}, {"headMentionIndex": 4, "tailMentionIndex": 19, "type": "further specification"}, {"headMentionIndex": 4, "tailMentionIndex": 14, "type": "uses"}]}