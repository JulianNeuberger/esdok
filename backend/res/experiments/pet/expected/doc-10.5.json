{"nodes": [{"id": "0", "name": "The MPON", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "1", "name": "requests", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "2", "name": "a device takeover bid", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "4", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "5", "name": "a tender for the equipment takeover", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "6", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "7", "name": "the order of the MPON", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "8", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "9", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, {"id": "10", "name": "the master data", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}], "edges": [{"id": "0", "source": {"id": "1", "name": "requests", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "a device takeover bid", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "1", "source": {"id": "1", "name": "requests", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "2", "source": {"id": "1", "name": "requests", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "0", "name": "The MPON", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "3", "source": {"id": "1", "name": "requests", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "4", "source": {"id": "4", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "a tender for the equipment takeover", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "5", "source": {"id": "4", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "6", "source": {"id": "4", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "0", "name": "The MPON", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "7", "source": {"id": "4", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "8", "source": {"id": "6", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "the order of the MPON", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "9", "source": {"id": "6", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "0", "name": "The MPON", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "10", "source": {"id": "6", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "11", "source": {"id": "6", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "8", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "12", "source": {"id": "8", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "the order of the MPON", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "13", "source": {"id": "8", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "14", "source": {"id": "8", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "9", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "15", "source": {"id": "9", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "10", "name": "the master data", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "16", "source": {"id": "9", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.5", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}]}